EntityService/TokenService.php

```diff
@@ -122,11 +123,16 @@ public function setToken(Token $newToken){
                     }
                     // If the channel has the same access token and the same scope,
                     // or no scope has been defined, then we're done.
-                    $status = self::STATUS_SAME_SCOPE;
 
-                    return $status;
+                    return self::STATUS_SAME_SCOPE;
+                } elseif ($newToken->getApplication()->getResourceOwner() === LinkedInController::RESOURCE_OWNER) {
+                    //Linkedin sends the same token
```

Comment:
Could you please be more elaborate in this comment, so that others can better understand the specific condition of a OAuth-based authentication where the channel sends back the same token? Feel free to use Linkedin as an example.